test/TensorFlowRuntime/tensor.swift

```diff
@@ -778,6 +778,25 @@ func testShapeGetter4() {
 }
 TensorTests.testAllBackends("ShapeGetter4", testShapeGetter4)
 
+@inline(never)
```

Comment:
Similarly, `@inline(never)` should be dropped and we should use the trailing closure syntax.

```swift
TensorTests.testAllBackends("replacing(with:where:)") {
   let original = Tensor<Float>([[1.0, 0.0], [3.0, 0.0], [2.0, 3.0], [1.0, 0.0]])
   let modified = original.replacing(with: Tensor<Float>(-1).broadcast(like: original), where: original.<3)
   let expected = Tensor<Float>([[-1.0, -1.0], [3.0, -1.0], [-1.0, 3.0], [-1.0, -1.0]])
   expectEqual(expected, modified)
}
```

Reply:
Also, could you please make sure these tests fit within 80 columns?